libs/sdk-ui/src/base/react/placeholders/tests/useComposedPlaceholder.test.tsx

```diff
@@ -29,13 +29,21 @@ const ComponentWithUsePlaceholderHook = (props: IComponentWithUsePlaceholderHook
 };
 
 interface IComponentWithResultProps {
-    result: object;
+    result: any;
```

Comment:
Isn't this possible to type somehow? It will be difficult to maintain the test if implementation changes.